Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -83,6 +85,8 @@ - (void)setUp
     [self invalidate];
     return;
   }
+  _usageMetricsTimer = [NSTimer scheduledTimerWithTimeInterval:1.0f
```

Comment:
preserving @tadeuzagallo comment:
> This timer is sad... It would probably be better to just include it on bridge calls, and set a longer timer in case the bridge is very idle. Also maybe a way to disable it? If you were profiling on chrome you probably wouldn't want it.